### ISSUE TITLE
Object.assign not supported in PhantomJS

### ISSUE THREAD

**Issue by ernsheong** [rdy/jasmine-async-suite#1]
Hi there, 

First of all, thanks for this.

We're getting an error in PhantomJS only (for CI) but it works fine in Chrome:

```
PhantomJS 2.1.1 (Windows 7 0.0.0) ERROR
  TypeError: undefined is not a constructor (evaluating 'Object.assign(global, _extends({}, withAsync(oldJasmineFns)))')
  at D:/workspace/class-direct/spec.bundle.js:64020 <- webpack:///~/jasmine-async-suite/index.js:34:0
```

Any idea or have a fix? Or is this a PhantomJS issue?

**Comment by ernsheong** [comment]
Not a library issue. PhantomJS does not have `Object.assign`

**Comment by rdy** [comment]
The transpiled code in babel is producing this accidentally. I'll make a change so that it polyfills it with an ES5 safe version.

**Comment by ernsheong** [comment]
Wow great, thanks!! :+1:

**Comment by rdy** [comment]
fixed in 0.0.6
